agent/proxycfg/manager_test.go

```diff
@@ -310,7 +309,7 @@ func testManager_BasicLifecycle(
 
 	require := require.New(t)
 
-	logger := log.New(os.Stderr, "", log.LstdFlags)
+	logger := testutil.TestHcLog(t.Name())
```

Comment:
This should ultimately funnel logs through a `testutil.TestWriter` and helpfully you already have a `t` in scope so it's not a difficult fix.